plugins/bulk-import/src/components/Repositories/DeleteRepositoryDialog.tsx

```diff
@@ -69,27 +70,33 @@ const DeleteRepositoryDialog = ({
   closeDialog: () => void;
 }) => {
   const classes = useStyles();
-  const [error, setError] = React.useState<string>('');
-  const [isSubmitting, setIsSubmitting] = React.useState(false);
+  const [deleteError, setDeleteError] = React.useState<string>('');
```

Comment:
useMutation returns an error. is it maybe simpler if you use `mutationDelete.error`. I would expect you can remove this error handling.